src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    private final String nameFigure = "Circle";
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public String getNameFigure() {
+        return nameFigure;
+    }
```

Comment:
Notice that you added `figureName` to each figure - why not moving it to `Figure` then?
Besides that, it's not necessary so you can remove it as well and just write the name in `draw()` method.